frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -293,6 +298,12 @@ export const surveyLogic = kea<surveyLogicType>([
                         },
                     }
                 },
+                setSurveyTemplateValues: (state, { template }) => {
+                    return {
+                        ...state,
```

Comment:
I think this _might_ be causing the borking w.r.t sporadic redirecting to existing survey.

This should override the survey, get rid of the id and any extra info added here.

Reply:
yes! i think you're correct here.